builder/lxd/config.go

```diff
@@ -42,6 +42,8 @@ type Config struct {
 	LaunchConfig map[string]string `mapstructure:"launch_config" required:"false"`
 	// Create LXD virtual-machine image on hosts running LXD 4.0 and above; defaults to false for container image
 	VirtualMachine bool `mapstructure:"virtual_machine"`
+	// Not execute `lxc publish`; defaults to false
+	NoPublish bool `mapstructure:"no_publish" required:"false"`
```

Comment:
For skipping build steps, we usually use `skip_*` which sounds a little better. 
```suggestion
	SkipPublish bool `mapstructure:"skip_publish" required:"false"`
```

Reply:
Thanks, I will change it.